src/test/java/com/example/demo/service/EmployeeServiceTest.java

```diff
@@ -221,7 +221,37 @@ public void givenTeamLeadId_deleteTeamLead_expectException() {
         when(employeeRepository.findById(1L)).thenReturn(Optional.empty());
 
         service.deleteTeamLeadId(employeeId);
+    }
+
+    @Test
+    public void givenTeamLeadId_getSubEmployees(){
+        EmployeeEntity employeeEntity = new EmployeeEntity();
+        employeeEntity.setId(1L);
+        employeeEntity.setJob("Gradinar");
+        EmployeeEntity employeeEntity1 = new EmployeeEntity();
```

Comment:
you can extract the creation of the employee into a method and avoid duplicate code